ci_config.json

```diff
@@ -190,6 +190,76 @@
       "libxkbcommon-dev"
     ]
   },
+  "fscl-xalgorithms-c": {
+    "fatal_warnings": true,
```

Comment:
Unnecessary: it already defaults to true.

Reply:
Assuming that's the norm removed extra flag fatal_warnings